Sources/SwerverCore/Response/HttpResponse.swift

```diff
@@ -11,9 +11,9 @@ public final class HttpResponse {
     public let allowedMethods: [String]
 
     init(builder: ResponseBuilder) {
-        self.statusCode = builder.statusCode
+        self.statusCode = builder.statusCode?.rawValue ?? String()
```

Comment:
`statusCode` should be stored as an enum on the `HttpResponse`. That gives you type safety if/when you are programatically creating responses.

This can be done in a future commit/PR

Reply:
noted in new trello card